demo/admin/src/dashboard/LatestBuilds.tsx

```diff
@@ -1,10 +1,19 @@
 import { gql } from "@apollo/client";
-import { Table, TableQuery, useTableQuery } from "@comet/admin";
-import { BuildRuntime } from "@comet/cms-admin";
+import { useTableQuery } from "@comet/admin";
+import { LatestBuildsDashboardWidget } from "@comet/cms-admin";
 import { GQLLatestBuildsQuery, GQLLatestBuildsQueryVariables } from "@src/graphql.generated";
-import { parseISO } from "date-fns";
 import * as React from "react";
-import { useIntl } from "react-intl";
+
+export const LatestBuilds: React.FC = () => {
+    const tableQuery = useTableQuery<GQLLatestBuildsQuery, GQLLatestBuildsQueryVariables>()(LATEST_BUILDS, {
```

Comment:
@nsams I think it would be a good time to change the widgets from Comet Admin Table to DataGrid in this PR, what do you think?

Reply:
sure, should be changed... (possibly out of scope for this PR)

the only important thing is that we don't have a prop tableQuery that needs useTableQuery, as that is deprecated and not really usable for DataGrid. Dashboard items don't need paging/filtering etc, so you can simply pass the data and we can switch the implementation in library at any time. (if things are simpler: now)

Reply:
@johnnyomair @nsams I will do this in a separate PR.

Reply:
@nsams ok, I created a task for myself for next week.